dockerfiles/platform-build/Dockerfile

```diff
@@ -20,8 +20,8 @@
 ARG BASE_VER=7.12
 FROM hpccsystems/platform-build-base:${BASE_VER}
 
-RUN groupadd -g 1000 hpcc
-RUN useradd -s /bin/bash -r -m -N -c "hpcc runtime User" -u 999 -g hpcc hpcc
+RUN groupadd -g 10001 hpcc
+RUN useradd -s /bin/bash -r -m -N -c "hpcc runtime User" -u 10000 -g hpcc hpcc
```

Comment:
0-999 typically reserved for system accounts.
Any reason this is 10K(+1) vs 1K(+1) ?

Reply:
Because that's what kube-score requires to consider it acceptable...